pkg/k8s/service.go

```diff
@@ -111,7 +111,7 @@ func ParseService(svc *types.Service) (ServiceID, *Service) {
 		// would introduce more complexity in already too complex LB codebase,
 		// so for now (until we have refactored the LB code) keep NodePort
 		// frontends in Service.NodePorts.
-		if svc.Spec.Type == v1.ServiceTypeNodePort {
+		if svc.Spec.Type == v1.ServiceTypeNodePort || svc.Spec.Type == v1.ServiceTypeLoadBalancer {
```

Comment:
Is ServiceTypeLoadBalancer always guaranteed to be within nodeport port range?

Reply:
Good question. I haven't checked the code, but it seems to allocate from the NodePort range.